red/runtime/nodes/flows/index.js

```
@@ -647,10 +777,16 @@ function removeFlow(id) {
     }
 
-    var newConfig = clone(activeConfig.flows);
-    newConfig = newConfig.filter(function(node) {
-        return node.z !== id && node.id !== id;
-    });
+    if (settings.enableFlowSharding) {
+        var newConfig = activeConfig.flows.filter(function(node) {
+            return node.z !== id && node.id !== id;
+        });
+    } else {
+        var newConfig = clone(activeConfig.flows);
```

Biome (2.1.2)

[error] Shouldn't redeclare 'newConfig'. Consider to delete it or rename it.

'newConfig' is defined here:


(lint/suspicious/noRedeclare)